sqlmesh/core/engine_adapter/base.py

```diff
@@ -1847,13 +1849,14 @@ def temp_table(
             finally:
                 self.drop_table(table)
 
-    def _table_properties_to_expressions(
-        self, table_properties: t.Optional[t.Dict[str, exp.Expression]] = None
+    def _model_properties_to_expressions(
```

Comment:
Model properties means something else throughout the code so let's not use it here. Additionally, the engine adapter is too low level of a construct to be aware of something as high-level as "model".

I'm ok keeping with keeping this as `table_properties` or just `properties` since that's what we create here. If you want to be more specific I'm also fine with something like `table_or_view_properties`.

Reply:
Ok I see what you mean!

The goal of that change was to be consistent with the call parameters since we have both:
- https://github.com/Kayrnt/sqlmesh/blob/8bae4b47b7bd5a038f05bd3d51a126c4c747fcf8/sqlmesh/core/engine_adapter/bigquery.py#L532
- https://github.com/Kayrnt/sqlmesh/blob/8bae4b47b7bd5a038f05bd3d51a126c4c747fcf8/sqlmesh/core/engine_adapter/bigquery.py#L571

I would also suggest for `table_or_view_properties` (and `table_or_view_properties_to_expressions`) then as I'd rather have future maintainers understand that the function expects one or the other (and it's not a bug).

I'll make an update